test/custom_class.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc
+; ModuleID = 'custom_class.ll'
+source_filename = "/localdisk2/icl/fadeeval/sycl_workspace/repo2/custom_class.cpp"
```

Comment:
Please, remove this line because it is useless for test and discloses unnecessary information.
Could you please include cpp test file contents in comments and command that you used for creation of this IR?
You can see an example in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/test/DebugInfo/X86/dwarf-aranges-no-dwarf-labels.ll#L14